### DISCUSSION TITLE
Nextcloud AIO automatically renewed let's encrypt certificate for on premise nextcloud web but failed to do the same for AIO certificate

### DISCUSSION THREAD

**Discussion by nuschpl** [nextcloud/all-in-one#1759] [Questions] [answered]
### Steps to reproduce
1. Install nextcloud aio along with guidelines
2. Wait a year
3. Have accessible nextcloud itself with current certificate but expired cert in AIO panel under port 8443

**Comment by szaimen** [comment]
Hi, can you check if port 80 is open/fowarded in your router?

**Comment by nuschpl** [comment]
it is not as I'm not interested in exposing non HTTPS services in 2023. If you mean Let's encrypt challenges then I preffer other methods. Right now I don't have access to service on TCP 8443 at all. I've logged in locally to AIO port 8080 , I've updated master container along with others and now when I click AIO link as admin from :443 panel it redirects me to local IP address and 8080 port instead of domain name and 8443 port as before. Is there way I can reeanble it ?

**Comment by szaimen** [comment, marked as answer]
Port 80 is needed for getting a valid certificate for the AIO interface at port 8443. As for changing the link of the button, see https://github.com/nextcloud/all-in-one#how-to-easily-log-in-to-the-aio-interface
